README.md

```diff
@@ -556,3 +558,27 @@ yarn start
 ## Stats
 ![npm bundle size](https://img.shields.io/bundlephobia/minzip/regular-table) 
 -->
+
+## Contributing
+For any questions, bugs or feature requests please open an [issue](https://github.com/finos/regular-table/issues)
+
+To submit a contribution:
+1. Fork it (<https://github.com/finos/regular-table/fork>)
+2. Create your feature branch (`git checkout -b feature/fooBar`)
+3. Read our [contribution guidelines](./CONTRIBUTING.md) and [Community Code of Conduct](https://www.finos.org/code-of-conduct)
+4. Commit your changes (`git commit -am 'Add some fooBar'`)
+5. Push to the branch (`git push origin feature/fooBar`)
+6. Create a new Pull Request
+
+_NOTE:_ Commits and pull requests to FINOS repositories will only be accepted from those contributors with an active, executed Individual Contributor License Agreement (ICLA) with FINOS OR who are covered under an existing and active Corporate Contribution License Agreement (CCLA) executed with FINOS. Commits from individuals not covered under an ICLA or CCLA will be flagged and blocked by the FINOS Clabot tool (or [EasyCLA](https://community.finos.org/docs/governance/Software-Projects/easycla)). Please note that some CCLAs require individuals/employees to be explicitly named on the CCLA.
+
+
+*Need an ICLA? Unsure if you are covered under an existing CCLA? Email [help@finos.org](mailto:help@finos.org)*
+
+## License
```

Comment:
This provides no additional useful information on the README while occupying additional screen space. 

Github already puts that on the screen:
<img width="372" alt="Screenshot 2023-01-01 at 3 16 19 PM" src="https://user-images.githubusercontent.com/3105306/210183531-22934a70-3e13-49e5-8c77-6cad1b615e6d.png">

and more information can be gleaned from the LICENSE file which has been the convention for...you guessed it, decades.